samples/grids/grid/state-persistence-about/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Sample | Ignite UI | Web Components | infragistics</title>
+    <meta charset="UTF-8" />
+
+    <link rel="shortcut icon" href="https://static.infragistics.com/xplatform/images/browsers/wc.png" >
+    <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Kanit&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium Web" />
+    <link rel="stylesheet" href="https://static.infragistics.com/xplatform/css/samples/shared.v8.css" />
+    <link rel="stylesheet" href="./about.css" type="text/css" />
+  </head>
+
+  <body>
+    <div id="root">
+      <div class="container sample">
+        <div style="text-align: center; width: 900px">
+            <br />
+            <span>By default navigating to the previous page, components will reinitialize as per their initial configuration, therefore the igcGrid will lose its state.</span>
+            <br />
+            <span>What our Sample Component does is reading the state from the `window.localStorage` object and applying the corresponding state in the `onload` window lifecycle.</span><br />
```

Comment:
The way this is formatted seems to be break the logic after the build copies the content to the samples browser dir and it throws an error:
![image](https://github.com/IgniteUI/igniteui-wc-examples/assets/10397980/2d3a5ccc-7e9c-4af4-948b-c36f80426a8e)
 
Removing the `window.localStorage`, `onload` formatting should fix it.